config/core/test-configs.yaml

```diff
@@ -357,6 +368,12 @@ test_plans:
   smc:
     rootfs: buildroot_baseline_ramdisk
 
+  tast:
+    rootfs: chrome_version
+    pattern: 'chromeos/new-chromeos-template.jinja2'
```

Comment:
This name doesn't seem better than `chromeos/tast.jinja2`

Reply:
updated